torch/_torch_docs.py

```diff
@@ -3157,6 +3157,54 @@ def merge_dicts(*dicts):
     tensor([ 0.,  2.,  1.,  0.])
 """.format(**common_args))
 
+add_docstr(torch.histogram,
+           r"""
+histogram(input, bins=10, weights=None, min=0, max=0, density=False) -> Tensor
+
+Computes the histogram of a tensor.
+
+Args:
+    {input}
+    bins (int or 1D Tensor): If int, defines the number of bins. If Tensor,
+        defines a monotonically increasing sequence of bin edges, allowing
+        for non-uniform bins. Default: 10
+    min (float): lower end of the range (inclusive), ignored if bins is a Tensor
```

Comment:
I'm curious to see how you describe the range and the computation of bin starts/stops when the bin edges aren't specified.

Reply:
The documentation file got borked somehow, pushing the right one now.